opba-banking-rest-impl/src/test/java/de/adorsys/opba/tppbankingapi/mapper/FacadeResponseBodyToRestBodyMapperTest.java

```diff
@@ -30,4 +54,30 @@ public void testMapping() {
         assertEquals(IBAN, restEntity.getAccounts().get(0).getIban());
         assertEquals(DELETED, restEntity.getAccounts().get(0).getStatus().name());
     }
+
+    @Test
+    @SneakyThrows
+    void accountsMapperTest() {
+        URL inputUrl = Resources.getResource("mapperTestFixtures/facade_to_rest_response_accounts_mapper_input.json");
```

Comment:
Avoid capital letters in project path - 'mapperTestFixtures'. Not all OS'es do work properly with them.

Reply:
ok